### ISSUE TITLE
prediction_results_train_set.csv serialization bug

### ISSUE THREAD

**Issue by bdecost** [usnistgov/alignn#88]
for single-output regression tasks, the predictions serialized to the datafile `prediction_results_train_set.csv` [in this block of code](https://github.com/usnistgov/alignn/blob/736ec739dfd697d64b1c2a01dc84678a24bcfacd/alignn/train.py#L1030) are not as expected

nominally this code should read predicted and target values from the EpochOutputStore and just write them to css
- by default this seems to serialize the validation set predictions, not training set predictions as implied by the file name
- there are two `f.write` statements, and the target and prediction values are swapped...

**Comment by bdecost** [comment]
IIRC the EpochOutputStore stores `(target, prediction)` so the second line for each validation set instance should be the correct one

to get the training predictions, read from `train_eos` instead of `eos` (`eos` stores the validation set outputs)

**Comment by bdecost** [comment]
resolved by #89
